posthog/celery.py

```diff
@@ -274,17 +274,20 @@ def recompute_materialized_columns_enabled() -> bool:
 @app.task(ignore_result=True)
 def clickhouse_materialize_columns():
     if recompute_materialized_columns_enabled():
-        from ee.clickhouse.materialized_columns.analyze import materialize_properties_task
+        from posthog.clickhouse.materialized_columns import materialize_properties_task
 
         materialize_properties_task()
 
 
 @app.task(ignore_result=True)
 def clickhouse_mark_all_materialized():
     if recompute_materialized_columns_enabled():
-        from ee.tasks.materialized_columns import mark_all_materialized
-
-        mark_all_materialized()
+        try:
```

Comment:
same question. This pattern seems weird. This task, if ever run on foss, will break.

So, move these to something like `posthog/ee/celery.py` ?

Reply:
hmm, yes